tests/test_transform.py

```diff
@@ -124,14 +130,7 @@ def test_search_space_transform_numerical(
 
     trans_params = trans.transform({"x0": param})
     assert trans_params.size == 1
-
-    if isinstance(
-        distribution,
-        (DiscreteUniformDistribution, IntUniformDistribution, IntLogUniformDistribution),
-    ):
-        assert expected_low <= trans_params <= expected_high
-    else:
-        assert expected_low <= trans_params < expected_high
```

Comment:
Even if a `distribution` is a real distribution, a transformer doesn't subtract eps from a upper bound thus it would be the same as `expected_high`.

https://github.com/optuna/optuna/runs/4506565155?check_suite_focus=true